src/hyperlight_host/src/sandbox/initialized_multi_use.rs

```diff
@@ -213,7 +195,7 @@ impl MultiUseSandbox {
                 args.into_value(),
             );
             let ret = Output::from_value(ret?);
-            self.mem_mgr.unwrap_mgr_mut().push_state()?;
+            self.mem_mgr.unwrap_mgr_mut().push_state().unwrap();
```

Comment:
Do we really want to panic here in case of an Err?
Shouldn't a `?` suffice?
```suggestion
            self.mem_mgr.unwrap_mgr_mut().push_state()?;
```

Reply:
I think that was a leftover from some debugging.
That line is removed by a later commit anyway.
Do you have a strong preference on fixing the intermediate commit?

Reply:
I don't have strong preferences about this. I just commented as I reviewed it.
Good thing that it got solved.